workers/unity/Packages/com.improbable.gdk.tools/.DownloadCoreSdk/Program.cs

```diff
@@ -0,0 +1,133 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Runtime.InteropServices;
+
+namespace Improbable
+{
+    internal class Program
+    {
+        private static void Main(string[] args)
+        {
+            if (args.Length == 0)
+            {
+                Console.Error.WriteLine("Specify the version of the CoreSdk to download.");
+                Environment.Exit(1);
+            }
+
+            var coreSdkVersion = args[0];
+            var nativeDependenciesPath = Path.GetFullPath("./Assets/Plugins/Improbable/Core");
+            var managedDependenciesPath = Path.GetFullPath("./Assets/Plugins/Improbable/Sdk");
+            var tempPath = Path.GetFullPath($"./build/CoreSdk/{coreSdkVersion}");
```

Comment:
Is there any reason not to use the Temp Unity directory?

Reply:
I originally did use `Temp`, Unity completely blows away it away when it shuts down.

Reply:
Ah right - that is no bueno then